app/src/main/java/com/futurice/freesound/feature/search/SearchFragmentViewModel.java

```diff
@@ -57,7 +57,7 @@
         return Observable.fromIterable(sounds)
                          .map(sound -> DisplayableItem.create(sound, SOUND))
                          .toList()
-                         .blockingFirst();
+                         .blockingGet();
```

Comment:
Keeping as is for now, if you don't mind. We can revisit in a separate PR if we feel strongly about it.